### DISCUSSION TITLE
Samsung Monitor S27R65X not detected

### DISCUSSION THREAD

**Discussion by shmuelsash** [xanderfrangos/twinkle-tray#486] [Monitor Compatibility Issues]
I have 2 exactly the same monitors. The plugged into DVI through an DVI -> HDMI adaptor works perfectly but the one plugged directly into HDMI is not working
However, when I switch the HDMI one into the DVI port then it does work
This makes me think there is something wrong with the HDMI port or some setting

Any suggestions? 
 
P.S. I uninstalled AMD Software it didn't help

PC Specs
AMD Ryzen 7 5700G with Radeon Graphics 3.80 GHz
Windows 11 Pro
Version 22H2

![image](https://user-images.githubusercontent.com/35056305/202484808-30a924ba-4c4d-46ce-9a4d-d6328625176b.png)

**Comment by shmuelsash** [comment]
With ClickMonitorDDC I get a drop of control on the HDMI screen but not full
This is the info pulled by the program for the DVI one
![Screenshot_20221117_115602](https://user-images.githubusercontent.com/35056305/202509303-33e8b72d-3f01-4bfd-a68e-346a97721145.png)

HDMI one
![Screenshot_20221117_115625](https://user-images.githubusercontent.com/35056305/202509346-86320724-ad2c-4423-815f-c489c2634eb6.png)

**Reply by shmuelsash** [reply]
I'm able to pull VCP codes from the DVI one but not the HDMI one

**Reply by xanderfrangos** [reply]
Sounds like an AMD driver issue or a monitor issue. AMD drivers seem to be the worst for DDC/CI. If it's not any better with ClickMonitorDDC, then there isn't anything I can suggest that you haven't already tried.
